biojava-structure/src/main/java/org/biojava/nbio/structure/basepairs/MismatchedBasePairParameters.java

```diff
@@ -0,0 +1,123 @@
+package org.biojava.nbio.structure.basepairs;
+
+import org.biojava.nbio.structure.Atom;
+import org.biojava.nbio.structure.Chain;
+import org.biojava.nbio.structure.Group;
+import org.biojava.nbio.structure.Structure;
+import org.biojava.nbio.structure.contact.Pair;
+
+import javax.vecmath.Matrix4d;
+import java.io.Serializable;
+import java.util.ArrayList;
+import java.util.List;
+
+/**
+ * Contributed to BioJava under its LGPL
+ * This class allows for finding inter-strand base pairs that are not necessarily canonical Watson-Crick pairs.
+ * The implementation of findPair is different than that of the base class.
+ * @author Luke Czapla
+ * @since 5.0.0-snapshot
```

Comment:
This should be `@since 5.0.0`. We don't use SNAPSHOTS in tags since those are transient builds changing every commit.